build.gradle

```diff
@@ -28,12 +15,21 @@ dependencies {
     implementation gradleApi()
     implementation localGroovy()
 
-    compileOnly "com.buschmais.jqassistant.cli:jqassistant-commandline-neo4jv3:1.7.0"
+    compileOnly "com.buschmais.jqassistant.cli:jqassistant-commandline-neo4jv3:1.9.1"
 
     testImplementation gradleTestKit()
     testImplementation('org.junit.jupiter:junit-jupiter:5.5.2')
 }
 
+test {
+    useJUnitPlatform()
+    // pass the proxy configuration to the gradle test executor
+    systemProperty "http.proxyHost", System.getProperty('http.proxyHost')
+    systemProperty "http.proxyPort", System.getProperty('http.proxyPort')
+    systemProperty "https.proxyHost", System.getProperty('https.proxyHost')
+    systemProperty "https.proxyPort", System.getProperty('https.proxyPort')
```

Comment:
@aews I'm curious, why do you think is this required?

Reply:
@f4lco without this Gradle is not using my proxy setting from the system environment variables I have configured. An alternative would be to use a gradle properties file, so you can remove it if you don't want it.

Reply:
OK thanks for the feedback. Yes, I'll remove it then, it really looks like host-specific configuration to me. Looks like you are already aware of [declaring system properties in gradle.properties](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_system_properties) and [the location of gradle.properties within the Gradle installation](https://docs.gradle.org/current/userguide/build_environment.html#sec:gradle_environment_variables). That would allow to use the same proxy settings regardless of the current Gradle project.